pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "up42-py"
 version = "0.33.1"
 description = "Python SDK for UP42, the geospatial marketplace and developer platform."
-authors = ["UP42 GmbH"]
+authors = ["UP42 GmbH <support@up42.com>"]
 license = "https://github.com/up42/up42-py/blob/master/LICENSE"
```

Comment:
We need to change pyproject, otherwise it doesn't have a static license which is problematic since we can change the licensing affecting old versions

Reply:
Setup.py just used MIT

Reply:
@Javid do you have a proposal for solution?
I do not want to add the entire [license string](https://github.com/up42/up42-py/blob/master/LICENSE) here.